pennylane/templates/embeddings/iqp.py

```diff
@@ -228,7 +228,7 @@ def compute_decomposition(
         op_list = []
         for _ in range(n_repeats):
 
-            for i in range(len(wires)):
+            for i in range(len(wires)):  # pylint: disable=consider-using-enumerate
```

Comment:
how come this was disabled?

Reply:
Makes sense!

Reply:
very very minor, but might make sense to leave a comment as to why `range` is used just so a future developer doesn't change it